lib/public/Calendar/Events/AbstractCalendarObjectEvent.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCP\Calendar\Events;
+
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IWebhookCompatibleEvent;
+
+/**
+ * @since 32.0.0
```

Comment:
```suggestion
 * @since 31.0.2
```

everywhere? or what is the current state of the `@since backporting` strategy